ConsumerApi.Tests.Integration/StepDefinitions/MessagesStepDefinitions.cs

```diff
@@ -55,26 +136,82 @@ public async Task WhenAPostRequestIsSentToTheMessagesEndpoint()
             [
                 new SendMessageRequestRecipientInformation
                 {
-                    Address = _client2.IdentityData!.Address,
+                    Address = recipient.IdentityData!.Address,
                     EncryptedKey = ConvertibleString.FromUtf8("AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA").BytesRepresentation
```

Comment:
We have `GetBytes()` extension method for this I believe

Reply:
It wasn't actually added by me, I think. But I will fix it. :)